### ISSUE TITLE
Consultation data volume

### ISSUE THREAD

**Issue by alwualwu** [om-ai-lab/OpenTrackVLA#13]
Hello, may I ask what size of data you are training on? I have generated approximately 2300 10 second MP4 files to train the STT task, but the final evaluation results were poor, so I would like to refer to your data size.Finally, thank you for your work!

**Comment by kyusonglee** [comment]
We use about 400K data points (roughly 400K lines in the JSONL). Did you filter out unsuccessful episodes?
Also, this work originally came from a closed project, and we later decided to open-source the code and data structure. Please understand that we’re not able to release the dataset at this time.
We’re currently focusing on building the data engine, which we believe is the key to this task. Once we make meaningful progress, we’ll revisit whether to open-source the data and/or the data engine.

**Comment by alwualwu** [comment]
Hello, thank you very much for your reply. My data size is around 170K, and I filtered out failed data during training. Perhaps it's because the amount of data I trained was too small or the training was insufficient, resulting in poor performance. I am currently working hard to collect more data. Can you tell me the epoch you trained for me to refer to. Thank you very much for your open-source work on OPenTrackVLA

**Comment by kyusonglee** [comment]
Could you share your training settings (learning rate, number of GPUs, batch size, etc.)? If the settings are correct, you should see a good trajectory shape in the training logs under the vis folder even after just one epoch on the 170K STT dataset. To achieve a higher success rate, you’ll need more data, but 170K should be enough to confirm the training pipeline is working. Also, make sure to filter out bad data before starting training.

**Comment by alwualwu** [comment]
Hello, I am using a 4090 GPU for training, and the training code is the same as yours: 
python train.exe\
--train_json data/sample/jsonl \
--cache_root data/sample/vision_cache \
--out_dir ckpt_sample \
--epochs 2 \
--batch_size 8 \
--n_waypoints 8 \
--history 31 \
--lr 2e-5 \
--mixed_precision \
However, when I evaluated the stt task using my trained model, the success rate was 37.03 and the tracking rate was 65.85. Perhaps this is because I only trained on the STT task? Here are some tracks from my vis folder.

![Image](https://github.com/user-attachments/assets/d0aff07f-79bd-4c06-b398-a89b58259bf8)
![Image](https://github.com/user-attachments/assets/55316420-96d1-476b-8489-f5735e7d1e15)
![Image](https://github.com/user-attachments/assets/4c628ffe-73a0-433d-9eb1-ee53afadd24b)

**Comment by kyusonglee** [comment]
It looks good. Here’s the training workflow I followed:
Train a model. Use that model to collect a new round of data. Filter and curate the collected data. Continue training by fine-tuning on top of the previously trained checkpoint. Repeat this cycle multiple times.
I also used a VLM to augment the data. I can’t share more implementation details because the data-pipeline method hasn’t been published yet, but this should be a solid starting point for you to build on and improve.

**Comment by duhuangzou** [comment]
> Hello, may I ask what size of data you are training on? I have generated approximately 2300 10 second MP4 files to train the STT task, but the final evaluation results were poor, so I would like to refer to your data size.Finally, thank you for your work!

Hello, sorry to bother you. May I ask how you generated the MP4 videos? When I run bash stt_data.sh, every episode fails, so nothing is generated.
